wire-ios/Wire-iOS/Sources/UserInterface/BlockerViewController.swift

```diff
@@ -43,6 +44,11 @@ final class BlockerViewController: LaunchImageViewController {
         fatalError("init?(coder aDecoder: NSCoder) is not implemented")
     }
 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupApplicationNotifications()
+    }
+
     override func viewDidAppear(_ animated: Bool) {
         showAlert()
```

Comment:
is it intended we don't call super here?

Reply:
fixed